src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -76,16 +76,16 @@
     (:postponed :pending)   (transaction-icon :icons/arrow-right styles/color-gray4-transparent styles/color-gray7)
     (throw (str "Unknown transaction type: " k))))
 
-(defn render-transaction [{:keys [hash to from type value symbol] :as transaction}]
+(defn render-transaction [{:keys [hash from-contact to-contact to from type value symbol] :as transaction}]
```

Comment:
`from-address` ?

Reply:
that's the contact either from-contact or to-contact contains the name of the contact.

Reply:
it is a slightly better terminology but `to` and `from` are the name of the fields in transaction, I think it is better to keep the same names for the same things